experiment_helpers/randomize_stimuli.m

```diff
@@ -26,8 +29,30 @@
 %       (or specifically, the first indexed condition in expt.conds),
 %       specify that number of trials here. If this isn't a multiple of
 %       expt.ntrials_per_block, you may have an uneven # of pert trials.
+% 
+%   bAlternatePnp       Default = 1 (to assume the original behavior of this script). binary "alternate perturbed and
+%                       non-perturbed"
+%                       This is a binary flag to indicate if you are doing an experiment where no perturbed trial can be next
+%                       to another perturbed trial (bAlternatePnp = 1) or if you are doing an experiment where you just don't
+%                       want two of the same word OR same perturbation to be next to each other (bAlternatePnp = 0). 
```

Comment:
Assuming this is an accurate reading of the behavior, I feel like this description is clearer (even though everything gets explained more later):
```
Default = 1. Binary flag "alternate perturbed and non-perturbed".
If 1, indicates that a perturbation trial should never be adjacent to a perturbation trial. "Perturbation
trials" are trials where the expt.conds value is greater than 1. Adjacent trials can use the same word.
If 0, indicates that a trial should never be adjacent to another trial with THE EXACT SAME
expt.conds OR expt.words value. That is, two "perturbation trials" CAN be adjacent,
so long as they have different perturbation values.